akka-remote/src/main/resources/reference.conf

```diff
@@ -258,6 +258,14 @@ akka {
     # Messages that were negatively acknowledged are always immediately
     # resent.
     resend-interval = 2 s
+    
+    # Maximum number of unacknowledged system messages that will be resent
+    # each 'resend-interval'. If you watch many (> 1000) remote actors you can
+    # increase this value to for example 600, but a too large limit (e.g. 10000)
+    # may flood the connection and might cause false failure detection to trigger.
+    # Test such a configuration by watching all actors at the same time and stop
+    # all watched actors at the same time.
+    resend-limit = 200
```

Comment:
I think the factor that we cannot know is the production environment cpu and network capacity.

The default of 200 msg per 2 s is probably too low, but I guess we want to err on that side.